experiments/download_experts.sh

```diff
@@ -4,5 +4,7 @@
 SCRIPT_DIR="$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd)"
 PROJECT_DIR="$(dirname ${SCRIPT_DIR})"
 EXPERT_MODELS_DIR=${PROJECT_DIR}/expert_models
+REWARD_MODELS_DIR=${PROJECT_DIR}/reward_models
```

Comment:
Consider moving all data files to a single subdirectory? So e.g. `data/expert_models`, `data/reward_models`. This will let you decouple the data from source code location more easily in the future: e.g. if you were doing distributed training, you might want to have a shared data output directory mounted via NFS on multiple workers. This would also simplify `.gitignore`, syncing and reduce directory clutter. Would make file names longer though.

Reply:
Good idea, I'm also restructuring tests/data/cartpole_0 to tests/data/expert_models/cartpole_0 to mirror this